src/Components/Trending/Trending.jsx

```diff
@@ -0,0 +1,65 @@
+import "./Trending.scss";
+
+const Trending = () => {
+  return (
+    <>
+      <div className="head">
+        <h1>GUIDANCE GRID</h1>
+      </div>
+      <div className="searchbar">
+        <input type="text" placeholder="Search blogs" />
+      </div>
+      <secttion className="trending">
+        <h3>Trending Articles</h3>
+        <div className="trend-art1">
+          <div className="card">
+            <p className="art_name">
+              {" "}
+              01 <img src="/Images/profile.png" alt="" /> GDSC in Code.
+            </p>
+            <p>Code is now available!</p> <p className="time">Nov 21, 10 mins read.</p>
+          </div>
```

Comment:
You can make this a component `TrendingCard` and pass the data as props and display it.

You can check the `ArticleCard` Component for more details.